Expecta+Snapshots.podspec

```diff
@@ -11,5 +11,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.source_files = 'EXPMatchers+FBSnapshotTest.{h,m}'
   s.frameworks   = 'Foundation', 'XCTest'
-  s.dependencies = ['Expecta', 'FBSnapshotTestCase']
+  s.dependency	 	 'FBSnapshotTestCase', '1.6'
+  s.dependency	 	 'Expecta', '0.3.2'
```

Comment:
It's a tabs vs. spaces problem, https://github.com/dblock/ios-snapshot-test-case-expecta/commit/947d466fb5715d52b47403307cd3f2aef38f9171.